.github/workflows/common-docker.yml

```diff
@@ -0,0 +1,107 @@
+name: Docker
```

Comment:
I wonder if it wouldn't be better to store the reusable workflows in a separate repository, for example in the https://github.com/keep-network/ci repo or in some new dedicated repo (like `keep-network/reusable-workflows`). We may want to use such workflows outside of `keep-network/keep-core` in the future. I know keeping them in `keep-core` repo doesn't prevent other repos from using them, but keeping them in dedicated place would seem more organized for me.

Reply:
We should extract it at some point. But for now, I prefer keeping it here to test and develop it easier. Once we confirm it works as expected we can extract it.